zombienet/0003-block-building-warp-sync/test-block-building-warp-sync.toml

```diff
@@ -0,0 +1,31 @@
+[settings]
+enable_tracing = false
+
+[relaychain]
+default_image = "{{ZOMBIENET_INTEGRATION_TEST_IMAGE}}"
+# default_command = "substrate"
+default_command = "/home/miszka/parity/07-fs-monitor/substrate/target/release/substrate"
+
+chain = "gen-db"
+chain_spec_path = "zombienet/0003-block-building-warp-sync/chain-spec.json"
+
+  [[relaychain.nodes]]
+  name = "alice"
+  validator = true
+  db_snapshot="https://storage.googleapis.com/zombienet-db-snaps/substrate/0001-basic-warp-sync/chains-0bb3f0be2ce41b5615b224215bcc8363aa0416a6.tgz"
+
+  [[relaychain.nodes]]
+  name = "bob"
+  validator = true
+  db_snapshot="https://storage.googleapis.com/zombienet-db-snaps/substrate/0001-basic-warp-sync/chains-0bb3f0be2ce41b5615b224215bcc8363aa0416a6.tgz"
+
+  #we need at least 3 nodes for warp sync
```

Comment:
suggestion: either move up or delete this comment. In the above `.toml` file there are 3 descriptions after the same comment, but, here, there are only two.

Reply:
Yep, it may be confusing. Actually this comment relates to third node, explaining why we need it.